{{cookiecutter.project_slug}}/deployment/application/base/deployment.yaml

```diff
@@ -30,19 +33,31 @@ spec:
         image: IMAGE
         ports:
         - name: uwsgi
-          containerPort: 8000
+          containerPort: {%- if cookiecutter.framework == 'SpringBoot' %} 8080 {%- else  %} 8000 {%- endif %}
         envFrom:
         - configMapRef:
             name: application
         - secretRef:
             name: application
         livenessProbe:
+          {%- if cookiecutter.framework == 'SpringBoot' %}
+          httpGet:
+            path: /actuator/health/liveness
+            port: uwsgi
+          {%- else %}
           tcpSocket:
             port: uwsgi
+          {%- endif %}
           initialDelaySeconds: 13
         readinessProbe:
+          {%- if cookiecutter.framework == 'SpringBoot' %}
+          httpGet:
+            path: /actuator/health/readiness
+            port: uwsgi
+          {%- else %}
           tcpSocket:
             port: uwsgi
+          {%- endif %}
```

Comment:
Just as a heads-up: We need to refactor this part.

1. Health checks (probes) should be generic, implemented for all frameworks.
1. This has nothing to do with "uwsgi" specifically here (only with Python).